src/modules/commentsCounter.js

```diff
@@ -0,0 +1,15 @@
+const commentsCounter = (data) => {
+  let counter = 0;
+  let commentHtml = '';
+  if (data) {
+    data.forEach((d) => {
+      commentHtml += `<p>${d.creation_date} ${d.username}: ${d.comment}</p>`;
+      counter += 1;
+    });
+    return counter;
+  }
+
+  return false;
+};
+
```

Comment:
- [ ] Similarly, let's ensure to indicate the comments counter value on the comment modal, like seen in the [template design](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md)

**Example of comment counter value on comment modal section:**
![image](https://user-images.githubusercontent.com/12745474/234874832-bfaaf565-129f-4d48-ad80-4240fa65fce6.png)

**Your comments modal:**
![image](https://user-images.githubusercontent.com/12745474/234875389-0bb17915-e51a-4853-8232-4bb7b8cd8558.png)